README.md

```diff
@@ -1,4 +1,40 @@
-bioformats-examples
-===================
+Bio-Formats examples
+====================
 
-Example code that demonstrates the Bio-Formats API in Java and C++
+This repository contains minimal examples of usage of Bio-Formats as a Java
+library using either Maven or Gradle.
+
+Prerequisites
+-------------
+
+The following are required:
+
+    JDK 7 or higher
+    Maven 2.x (for Maven build)
+
+Building with Maven
+-------------------
+
+From the root directory, run:
+
+    mvn
```

Comment:
The project builds fine when a goal is specified but not by default. Either the doc could be updated to include a suitable goal  (mvn clean install) or a default goal could be added to the pom eg <defaultGoal>install</defaultGoal>